src/main/webapp/WEB-INF/classes/log4j2.properties

```
@@ -1,42 +1,164 @@
-### Direct log messages to stdout
-log4j.appender.stdout=org.apache.log4j.ConsoleAppender
-log4j.appender.stdout.Target=System.out
-#log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
-log4j.appender.stdout.layout=com.celements.log4j.NoStackTracePatternLayout
-log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} [%X{url}] [%t] %-5p %-30.30c{2} %x - %m %n
-
-log4j.appender.file=org.apache.log4j.RollingFileAppender
-log4j.appender.file.File=${log4j.properties.appender.file.File}
-log4j.appender.file.MaxFileSize=10MB
-log4j.appender.file.MaxBackupIndex=50
-log4j.appender.file.layout=org.apache.log4j.PatternLayout
-log4j.appender.file.layout.ConversionPattern=%d{ISO8601} [%X{url}] [%t] %-5p %-30.30c{2} %x - %m %n
+status=error
+dest=err
+name=PropertiesConfig
+monitorInterval=${log4j.properties.monitorInterval}
 
-### By default everything that is of warning or severity WARN, ERROR or FATAL is logged both to
-### the console and to the xwiki.log file.
-log4j.rootLogger=warn, stdout, file
+### stdout appender
+appender.stdout.type=Console
+appender.stdout.name=stdout
+appender.stdout.target=SYSTEM_OUT
+appender.stdout.layout.type=PatternLayout
+appender.stdout.layout.pattern=%d{ISO8601} [%X{url}] [%t] %-5p %-30.30c{2} %x - %m %n
+
+## main appender
+appender.main.type=RollingFile
+appender.main.name=main
+appender.main.fileName=${log4j.properties.logdir}/celements.log
```

see comment on other PR
```suggestion
appender.main.fileName=${log4j.properties.logdir}/main.log
```